assets/tests/helpers/fullStory.test.ts

```diff
@@ -1,23 +1,45 @@
-import { describe, test, expect } from "@jest/globals"
-import { fullStoryIdentify } from "../../src/helpers/fullStory"
-import { mockFullStoryEvent } from "../testHelpers/mockHelpers"
+import { describe, test, expect, jest } from "@jest/globals"
+import { fullStoryInit } from "../../src/helpers/fullStory"
+import * as FullStory from "@fullstory/browser"
 
-describe("fullStoryIdentify", () => {
-  test("calls identify if username is given", () => {
-    mockFullStoryEvent()
+jest.mock("@fullstory/browser")
 
-    fullStoryIdentify("username")
+describe("fullStoryInit", () => {
+  test("calls init and identify if organization ID and username are given", () => {
+    const mockedFS = jest.mocked(FullStory)
 
-    expect(window.FS!.identify).toHaveBeenCalledWith("username", {
+    mockedFS.init.mockImplementationOnce(() => {})
+    mockedFS.identify.mockImplementationOnce(() => {})
```

Comment:
question(non-blocking): what does jest/fs do without these being explicitly implemented?

Reply:
Oh, that's a good question - because of the `jest.mock("@fullstory/browser")`, I think they're already effectively mocked out with no-op functions. If I remove the `mockImplementationOnce` calls the tests still pass. On the other hand, if I also remove the `jest.mock("@fullstory/browser")` call, the real FullStory init function tries to run and throws an error because of some aspects of the test environment that aren't to its liking.

Reply:
Explicit mocks removed in d6c7f385.